modules/raft-client/src/main/java/org/apache/ignite/raft/client/message/RaftClientMessageGroup.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.raft.client.message;
+
+import org.apache.ignite.network.annotations.MessageGroup;
+
+/**
+ * Message types for the raft-client module.
+ */
+@MessageGroup(groupType = 2, groupName = "RaftClientMessages")
+public class RaftClientMessageGroup {
+    /**
+     * Message type for {@link ActionRequest}.
+     */
+    public static final short ACTION_REQUEST = 1;
```

Comment:
Why don't we start with 0?

Reply:
I don't think it really matters, but I'll change that, if you like =)